### PR TITLE
Made examples of biorbd-viz work again

### PR DESCRIPTION
**Pull request by pariterre**
There were some issues regarding the shape of some elements in RotoTrans following major changes in the past months. They were fixed.

I also fixed a probable issue with get_frame, we must confirm though that you agree with the changes

### CODE REVIEW

File: pyomeca/frame_dependent.py [hunk 1]
```diff
@@ -674,7 +674,7 @@ def to_numpy(self):
 
     # --- Plot method
 
-    def plot(self, x=None, ax=None, fmt="", lw=1, label=None, alpha=1):
+    def plot(self, *args, x=None, ax=None, **kwargs):
         """
         Plot a pyomeca vector3d (Markers3d, Analogs3d, etc.)
```
Comment (romainmartinez) [inline pyomeca/frame_dependent.py]: We could explicitly say that *args and **kwargs are send to matplotlib's `plot` in the documentation

**Comment by romainmartinez** [comment]
Run `black .` in the root folder and we are good to go!

**Comment by pariterre** [comment]
Done :)

**Comment by pariterre** [comment]
@romainmartinez Do you approve this PRrrrr :D

**Comment by romainmartinez** [comment]
Yes I do!

**Pull request merged**